.github/actions/download-plugin/action.yml

```diff
@@ -0,0 +1,53 @@
+name: 'Download Security Plugin'
+description: 'Installs OpenSearch Dashboard with a Plugin from github, then checkouts the correct dashboards version for the plugin, configures npm/yarn, and bootstraps Dashboards'
+
+inputs:
+  opensearch-version:
+    description: 'The version of OpenSearch that should be used, e.g "3.0.0"'
+    required: true
+
+  plugin-name:
+    description: 'The the name of the plugin to use, such as opensearch-security'
+    required: true
+
+  plugin-version:
+    description: 'The version of security plugin that should be used, e.g "3.0.0.0"'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Download OpenSearch for Linux
+      uses: peternied/download-file@v2
+      if: ${{ runner.os == 'Linux' }}
+      with:
+        url: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{ inputs.opensearch-version }}/7028/linux/x64/tar/builds/opensearch/plugins/${{ inputs.plugin-name }}-${{ inputs.plugin-version }}.zip
```

Comment:
Yes, I was checking with infra team this morning and seems like the `latest` doesnt include security plugin yet. I will change it back to `latest` once everything is ready. This is only for now.

Reply:
Please link the issue where this is being worked on.

Reply:
I just linked issue in the general comment section.

Reply:
> Since the latest distribution build of 3.0.0 does not have security plugin included, so I'm testing on build version 7028 which includes all the latest changes we made for core and security plugin, and I will change it back to latest once this issue gets fixed [opensearch-project/cross-cluster-replication#696](https://github.com/opensearch-project/cross-cluster-replication/issues/696)

If the security plugin is building on 3.0.0 why don't we have a fresh build of it available?  Should we be publishing our own builds to maven for consumption so we aren't stuck because of a separate plugin, or should the build process automatically publish binaries for components that do function?

Reply:
@peternied here we go https://github.com/opensearch-project/opensearch-build/issues/3172

Reply:
Update: They have fixed the build